src/article/presentation/article.read.adapter.ts

```
@@ -27,4 +27,11 @@ export class ArticleReadAdapter {
 
         return new ResponseArticleForm(articleList);
     }
+
+    @Post('validate')
+    async validateItems(@Body() body: { organId: number; ids: number[] }): Promise<Boolean> {
```

Biome (1.9.4)

[error] Don't use 'Boolean' as a type.

Use lowercase primitives for consistency.
Safe fix: Use 'boolean' instead


(lint/complexity/noBannedTypes)